### PR TITLE
Footer is no longer fixed at bottom, instead it is at the very bottom

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -57,9 +56,12 @@ $footerHeight : 60px;
 footer {
   width: 100%;
   height: $footerHeight;
-  line-height: $footerHeight;
```
Comment (abobwhite) [inline app/styles/main.scss]: Why did you remove line-height? This existed so the content in the footer was vertically centered...

**Comment by ctwoolsey** [comment]
Yes, I knew why the line height was there.  I don't remember exactly what
the problem was, but the line-height was causing an extra margin at the
bottom of the page or some odd behavior which was fixed by removing it.

**Comment by ctwoolsey** [comment]
[image: Inline image 1]
This is what was happening with the line-height set.  Removing line-height
fixed this and so I didn't bother looking for another solution.

**Comment by ctwoolsey** [comment]
image

**Comment by abobwhite** [comment]
Why also did you remove `display: table-cell`? The combination of your css changes could have made the `line-height` an issue.

Is everything still vertically aligned to the middle?

**Comment by ctwoolsey** [comment]
Yes everything is vertically aligned to the middle.  I don't remember why I removed the Table-cell.  I cleaned up the footer code and the table-cell didn't seem to be doing anything if I remember correctly.  Or if it was doing something, it was breaking something else.  But the footer should look nice and clean with the text vertically centered. I added a padding to the top of the footer and the text looks nicely centered and scales.

**Comment by ctwoolsey** [comment]
When I make the line height equal to the footer height it has this problem.  I could make the line-height equal to 50px instead of the 60px (the footer height) and then I could remove the 15px padding on the footer-top.  And it still looks good and this mysterious bottom margin then disappears...Or I could leave it like it is because it looks good and works.

**Comment by ctwoolsey** [comment]
Here is how it looks currently:
image

**Comment by ctwoolsey** [comment]
Actually, Let me look at this again, because I'm seeing some issue when the page is smaller, that I didn't see before.

**Comment by abobwhite** [comment]
@ctwoolsey Please merge develop into this branch since it currently has a merge conflict with footer.html

**Comment by ctwoolsey** [comment]
OK, this should be ready to go now.

**Comment by abobwhite** [comment]
@ctwoolsey This is still showing 13 files changes over 5 commits. You only changed 3 files in your branch.

**Comment by ctwoolsey** [comment]
Yes, somehow it did that, but I did a revert on those commits.  So it really only affects the proper files.  Not sure how I can get rid of the other fIles unless I try creating a new branch and delete this one.

**Comment by abobwhite** [comment]
Something will have to be done. Because these are not changes you made in this branch and they should not show up here. Maybe a new branch might be the easiest without us looking at it.

**Comment by abobwhite** [comment]
Closed in favor of #25
